google/cloud/bigtable/tests/instance_admin_async_integration_test.cc

```diff
@@ -124,6 +133,67 @@ TEST_F(InstanceAdminAsyncIntegrationTest, AsyncCreateListDeleteInstanceTest) {
   pool.join();
 }
 
+/// @test Verify that cluster async CRUD operations work as expected.
+TEST_F(InstanceAdminAsyncIntegrationTest, AsyncCreateListDeleteClusterTest) {
+  std::string id =
+      "it-" + google::cloud::internal::Sample(
+                  generator_, 8, "abcdefghijklmnopqrstuvwxyz0123456789");
+  std::string cluster_id_str = id + "-cl2";
+
+  // create instance prerequisites for cluster operations
+  bigtable::InstanceId instance_id(id);
+  auto instance_config = IntegrationTestConfig(
+      id, "us-central1-f", bigtable::InstanceConfig::PRODUCTION, 3);
```

Comment:
Why the hard-coded zone? Please change this to receive the zone ids as command-line arguments.